guides/11_other-tutorials/create-immersive-demo.md

```diff
@@ -0,0 +1,199 @@
+# Create a Real-Time Immersive Audio + Video Demo with FastRTC
+
+Tags: REAL-TIME, IMMERSIVE, FASTRTC, VIDEO, AUDIO, STREAMING, GEMINI, WEBRTC
+
+FastRTC is a library that lets you build low-latency real-time apps over WebRTC. In this guide, you’ll implement a fun demo where Gemini is an art critic and will critique your uploaded artwork:
+- Streams your webcam and microphone to a Gemini real-time session
+- Sends periodic video frames (and an optional uploaded image) to the model
+- Streams back the model’s audio responses in real time
+- Creates a polished full-screen Gradio `WebRTC` UI
+
+### What you’ll build
+<gradio-app space="gradio/Gemini-Art-Critic"></gradio-app>
```

Comment:
Maybe a video here where it gives a funny answer is more engaging here and then we place the link/embed in the conclusion? No strong preference though